wagtail_localize_rws_languagecloud/management/commands/update_translated_pages.py

```diff
@@ -0,0 +1,128 @@
+            page_pk=Subquery(pages.values("id")[:1]),
+            page_title=Subquery(pages.values("title")[:1]),
+        ).filter(
+            # Only sources for live pages
+            object_id__in=Page.objects.live().values_list("translation_key", flat=True),
+            # Only pages that have been published after the last sync
+            last_updated_at__lt=F("last_published_at"),
+        )
+
+        sources_to_update = all_sources.exclude(
+            # Only pages without ongoing translation projects,
+            languagecloudproject__lc_project_status__in=[
+                LanguageCloudStatus.CREATED,
+                LanguageCloudStatus.IN_PROGRESS,
+            ]
+        )
+
+        total_count = len(all_sources)
+        to_update_count = len(sources_to_update)
+
+        logger.info(
+            f"Found {total_count} page(s) with stale translated content.\n"
+            f"       {to_update_count} will be synced.\n"
+            f"       {total_count - to_update_count} will be skipped."
+        )
+
+        all_pages = {(source.page_pk, source.page_title) for source in all_sources}
+
+        updated_pages = set()
+
+        for index, source in enumerate(sources_to_update):
+            updated_pages.add((source.page_pk, source.page_title))
+
+            counter = f"[{index + 1}/{to_update_count}]"
+
+            if options["dry_run"]:
+                logger.info(
+                    f'{counter} Would update translations for page: {source.page_pk} - "{source.page_title}"'
+                )
+                continue
+
+            logger.info(
+                f'{counter} Updating translations for page: {source.page_pk} - "{source.page_title}"'
+            )
+
+            # Sync content to translated pages
+            source.update_from_db()
+
+            enabled_translations = source.translations.filter(enabled=True)
+
+            # Prep for sending to RWS LanguageCloud
+            LanguageCloudProjectSettings.get_or_create_from_source_and_translation_data(
+                translation_source=source,
+                translations=enabled_translations,
+                name=LanguageCloudProjectSettingsForm.default_project_name_prefix,
+                description="Wagtail\n"
+                + LanguageCloudProjectSettingsForm._get_default_project_description(
+                    None, source
+                ),
+                due_date=LanguageCloudProjectSettingsForm.default_due_date,
+                template_id=LanguageCloudProjectSettingsForm.default_project_template_id,
+            )
+
+        skipped_pages = all_pages - updated_pages
+
+        for page_pk, page_title in skipped_pages:
+            logger.debug(f'Skipped page: {page_pk} - "{page_title}"')
+
+        if not options["dry_run"]:
+            logger.info("Sending summary emails")
+            send_update_translated_pages_emails(
+                list(all_pages), list(updated_pages), list(skipped_pages)
+            )
```

Comment:
This should respect the `SEND_EMAILS` setting (it is useful to be able to turn this off - e.g: in a staging or dev environment) - same as the other emails.
```py
if settings.WAGTAILLOCALIZE_RWS_LANGUAGECLOUD.get("SEND_EMAILS", False) and not options["dry_run"]:
    logger.info("Sending summary emails")
    send_update_translated_pages_emails(
        list(all_pages), list(updated_pages), list(skipped_pages)
    )
```

Reply:
Nice catch. I will add this.

Reply:
I've added this